---
layout: page
title: Lewis Quayle
excerpt: "More about Lewis..."
---

<p style="text-align:center;">
  <img src="/images/lquayle.png" width="425">
</p>

- GitHub: <a href="https://github.com/lquayle88" target="_blank">github.com/lquayle88</a>
- LinkedIn: <a href="https://www.linkedin.com/in/lewis-quayle" target="_blank">linkedin.com/in/lewis-quayle</a>
- Blog: <a href="https://www.lewisdoesdata.com" target="_blank">www.lewisdoesdata.com</a>
- YouTube: <a href="https://www.youtube.com/@lewisdoesdata" target="_blank">www.youtube.com/@lewisdoesdata</a>

## Skills

* Programming:
  
  * R, Python, SQL, Unix (BASh)
  * Markdown, Git, Nextflow, Docker, Singularity (Apptainer)

* Pipeline development and deployment on high-performance computing (HPC) or cloud-based infrastructure
* Microarray, RNA-Seq (bulk and single-cell), Whole Genome (WGS) and Whole Exome (WES) sequencing data analysis
* Relational database access and analysis
* Large-scale public and clinical data cleaning and integration
* Time-to-event (survival) analysis
* Interaction network construction, validation and analysis
* Geospatial data analysis
* Unsupervised and supervised machine learning model development and deployment
* Advanced data visualisation
* Dashboard (static and interactive) and web app development

## Experience

<table>
<tbody>
  <tr>
    <td>2022 - Present:</td>
    <td>Cancer Bioinformatician - Bioinformatics Core Facility, University of Sheffield</td>
  </tr>
  <tr>
    <td>2019 – 2022:</td>
    <td>Postdoctoral Bioinformatician - Medical School, University of Sheffield</td>
  </tr>
  <tr>
    <td>2017 – 2019:</td>
    <td>Postdoctoral Research Associate - Medical School, University of Sheffield</td>
  </tr>
  <tr>
    <td>2014 – 2017:</td>
    <td>PhD Medicine - Medical School, University of Sheffield</td>
  </tr>
  <tr>
    <td>2013 – 2014:</td>
    <td>MRes Medical & Biomedical Science - University of Lincoln</td>
  </tr>
  <tr>
    <td>2010 – 2013:</td>
    <td>BSc (Hons) Biomedical Science - University of Lincoln</td>
  </tr>
</tbody>
</table>

## Biography

I obtained my PhD in the Bone Oncology group of <a href="https://www.sheffield.ac.uk/medicine/people/oncology-metabolism/ingunn-holen" target="_blank">Prof. Ingunn Holen</a> and <a href="https://www.sheffield.ac.uk/medicine/people/oncology-metabolism/penelope-d-ottewell" target="_blank">Dr Penelope Ottewell</a> in the Department of Oncology & Metabolism at The University of Sheffield Medical School. My PhD project focussed on developing novel models of mitotic quiescence in human breast cancer cells *in vitro* and metastatic dormancy *in vivo*. My early postdoctoral work was focussed on characterising the cellular sub-populations isolated from these models using a multitude of approaches, including next-generation sequencing. During 2019, I undertook a three-month training placement in high-performance computing (HPC) and bioinformatic analysis of RNA-Seq data at the <a href="https://www.garvan.org.au/research/kinghorn-centre-for-clinical-genomics" target="_blank">Kinghorn Centre for Clinical Genomics</a> at the Garvan Institute of Medical Research in Sydney, Australia working under <a href="https://www.garvan.org.au/about-us/people/nenbar" target="_blank">Dr Nenad Bartonicek</a>. Upon my return to The University of Sheffield, I transitioned to a two-year position as a postdoctoral cancer bioinformatician and in June 2022 moved to my current role as cancer bioinformatician embedded within the Sheffield Bioinformatics Core Facility and working in the groups of <a href="https://www.sheffield.ac.uk/medicine/people/oncology-metabolism/james-catto" target="_blank">Prof. James Catto</a> and <a href="https://www.sheffield.ac.uk/medicine/people/neuroscience/dennis-wang" target="_blank">Dr Dennis Wang</a>.

My current research activity is centred on a project aiming to create a mutational map of the non-coding regions of patient genomes for the bladder cancer arm of the 100,000 Genomes Project. I am also contributing bioinformatics capability to the Genotype of Urothelial Cancer - Stratified Treatment and Oncological outcomes (GUSTO) phase II clinical study. These roles involve contributing to the maintenance of well-curated, highly structured, transparent resources for integrating genomic and clinical data, and assessing, testing, and implementing best practice scalable HPC workflows and methods for genomic data analysis.

My role within the Sheffield Bioinformatics Core Facility involves delivery of custom analyses, processed data, experimental results, and reports detailing quality control procedures and workflows carried out in secure containerised environments to core facility users. I am also involved in supporting the provision of best-practice training and workshops in next-generation sequencing and bioinformatics techniques to laboratory scientists, assisting investigators across the university in statistically sound experimental design, aiding interpretation of results, and grant or manuscript writing where these require extensive bioinformatics input.

## Publications

**Original Manuscripts**

Catto, J.W.F., Mandrik, O., **Quayle, L.A.**, Hussain, S.A., McGrath, J., Cresswell, J., Birtle, A.J., Jones, R.J., Mariappan, P., Makaroff, L.E., Knight, A., Mostafid, H., Chilcott, J., Sasieni, P., Cumberbatch M. (2023). Diagnosis, treatment and survival from bladder, upper urinary tract and urethral cancers: Real world findings from NHS England between 2013 and 2019. BJU Int. Accepted Author Manuscript. https://doi.org/10.1111/bju.15970

**Quayle, L.A.**, Spicer, A., Ottewell, P.D., Holen, I. (2021). Transcriptomic Profiling Reveals Novel Candidate Genes and Signaling Programs in Breast Cancer Quiescence and Dormancy. Cancers, 13(16): 3922.

Tulotta, C., Lefley, D., Moore,. C., Amariutei, A., Spicer, A., **Quayle, L.**, Hughes, R., Ahmed, K., Cookson, V., Evans, C., Vadakekolathu, J., Heath, P., Francis, F., Pinteaux, E., Pockley, A., Ottewell, P.D. (2021). IL-1B drives opposing responses in primary tumours and bone metastases; harnessing combination therapies to improve outcome in breast cancer. NPJ Breast Cancer, 7(1): 95.

**Quayle, L.A.**, Ottewell, P.D., Holen, I. (2018). Chemotherapy Resistance and Stemness in Mitotically Quiescent Human Breast Cancer Cells Identified by Fluorescent Dye Retention. Clin Exp Metastasis, 35(8): 831-846.

**Quayle, L.A.**, Pereira, M.G., Scheper, G., Wiltshire, T., Peake, R.E., Hussain, I., Rea, C.A. & Bates, T.E. (2017). Anti-angiogenic drugs: direct anti-cancer agents with mitochondrial mechanisms of action. Oncotarget, 8 (51): 88670-88688.

**Quayle, L.**, Ottewell, P.D., Holen, I. (2015). Bone Metastasis: Molecular Mechanisms Implicated in Tumour Cell Dormancy in Breast and Prostate Cancer. Curr Cancer Drug Targets. 15: 469-480.

**Book Chapters**

Holen, I. and **Quayle, L.A.** (2020). Stem cell niches in bone and their roles in cancer metastasis. Advances in Stem Cells and their Niches. Elsevier. 5: 35-62.

Ottewell, P.D. and **Quayle, L.A.** (2019) Tumor Dormancy in the Bone. Encyclopaedia of Bone Biology. Elsevier. 166 - 179.

**Conference Proceedings**

**Quayle, L.**, Ottewell, P.D., Holen, I. (2018), Therapeutic Resistance and Stemness in Mitotically Quiescent Human Breast Cancer Cells. 1st UK Interdisciplinary Breast Cancer Symposium 15th – 16th January 2018; Manchester, U.K. Breast Cancer Res Treat. 167:309–405: Abstract P10.12.

**Quayle, L.**, Park, S., McDonnell, D.P., Ottewell, P.D., Holen, I. (2017) Targeting ERR-α Regulated Lactate Metabolism Eliminates Drug-Resistant Breast Cancer Cells. Proceedings of the 2016 San Antonio Breast Cancer Symposium 6th – 10th December 2016; San Antonio, TX, U.S.A. Cancer Res. 77(4 Suppl): Abstract P3-07-14.

