---
layout: page
title: Talks, software and useful links
excerpt: "An archive of resources sorted by date."

---

## Talks

Talks and presentations given by members of the Core. For training materials and longer courses, see our [training](https://sbc.shef.ac.uk//training/) section.

- [Reproducible Bioinformatics pipelines with Nextflow](https://docs.google.com/presentation/d/17vXUbXOEzG6ZanTT-7tBqWVhLrxSUX3zv2hAvGk-Y_I/edit?usp=sharing), Mark Dunning, Northern BUG7 - Bradford
- [SARS-CoV-2 subgenomic RNA Detection](https://www.youtube.com/watch?v=aK8zp57FIK0), Matthew Parker, HHMI COVID-19 Seminar Series
- [periscope: Detecting subgenomic RNA in ARTIC Network Sequencing Data](https://nanoporetech.com/resource-centre/videos/NCM2020/Periscope%3A-sub-genomic-RNA-identification-in-SARS-CoV-2-ARTIC-Network-Nanopore-Sequencing-Data), Matthew Parker, Nanopore Community Meeting
- [Nanopore Sequencing of C9orf72 in ALS](https://nanoporetech.com/resource-centre/nanopore-sequencing-c9orf72-als), Matthew Parker, Nanopore London Calling
- [Statistical and Data Analysis Challenges in Bioinformatics](https://docs.google.com/presentation/d/1QPExPGWqQZJYZKPkQeEjVF-p_RvuRPdK17HFmWv91So/edit?usp=sharing), Local RSS group meeting, February 27th 2020
- [Data Formatting Introduction](https://sbc.shef.ac.uk/data-formatting/slides), BTEC Applied Science visit, June 14th 2019
- [Best Practices for teaching (and learning) R](https://markdunning.github.io/teaching-best-practice/slides.html#1), April 2nd 2019
- [Measuring Mutation Burden and Sequence Level Bias: An Initial  Assessment of 100K data](https://www.youtube.com/watch?v=O_aaNUTwkB8), 6th December 2018
- [Downstream Functional Analysis](https://drive.google.com/file/d/1BnmOms_pSJe00Br6ywkzvbJfL4rQYmXA/view?usp=sharing), Delivered as part of the MSC in Molecular Medicine, University of Sheffield, 8th March 2018
- [Overview of the Docker Container system](https://markdunning.github.io/docker-overview/), Talk at [SheffieldR](https://www.meetup.com/SheffieldR-Sheffield-R-Users-Group/), 6th March 2018


## Github

On our [github repository](https://github.com/sheffield-bioinformatics-core) you will find an index of bits of code we have written, published analyses we have performed for collaborators, training materials and even the code required to run this website. 

## Software

- Periscope; A tool to quantify sub-genomic RNA (sgRNA) expression in SARS-CoV-2 artic network amplicon nanopore sequencing data.
  + [github repository](https://github.com/sheffield-bioinformatics-core/periscope)


## Collaborators

Other researchers / Core facilities that we collaborate with and share resources

### Local

- [SITraN Multiomics Facility](http://sitran.org/facilities/genomics-sequencing/)
- [Sheffield Research Software Engineering (RSE) group](https://rse.shef.ac.uk/)
- [Sheffield Diagnostic Genetics Service](https://www.sheffieldchildrens.nhs.uk/sdgs/)
- [Sheffield Biological Mass Spectrometry Facility (biOMICS)](https://www.sheffield.ac.uk/biomics)
- [The NERC Biomolecular Analysis Facility - Sheffield](https://www.sheffield.ac.uk/nbaf-s/home)


### National / International

- [Northern Bioinformatics User Group (NorthernBUG)](https://northernbug.github.io/)
- [Cancer Research Uk Cambridge Institute Bioinformatics Core](https://www.cruk.cam.ac.uk/core-facilities/bioinformatics-core)
- [Bioinformatics Training Program of the University of Cambridge](https://bioinfotraining.bio.cam.ac.uk/)
- [Rockefeller University Bioinformatics](https://rockefelleruniversity.github.io/)





